src/layouts/ha-init-page.ts

```diff
@@ -54,10 +49,15 @@ class HaInitPage extends LitElement {
     }
   }
 
+  protected updated(changedProperties: PropertyValues) {
```

Comment:
Should start the import before rendering, no?

```suggestion
  protected willUpdate(changedProperties: PropertyValues<this>) {
```